yw_website/yw_website/apps/website/static/design/common.css

```diff
@@ -59,3 +59,3547 @@ body {
 .filler {
     height: .1em;
 }
+@charset "UTF-8";
```

Comment:
if we're copy and pasting css we should say where our source is

Reply:
done